src/h2load.cc

```diff
@@ -691,6 +707,18 @@ void Client::record_request_time(RequestStat *req_stat) {
   req_stat->request_time = std::chrono::steady_clock::now();
 }
 
+void Client::record_start_time(Stats *stat) {
+  stat->start_time = std::chrono::steady_clock::now();
+}
+
+void Client::record_connect_time(Stats *stat) {
+  stat->connect_time = std::chrono::steady_clock::now();
+}
+
+void Client::record_time_to_first_byte(Stats *stat) {
+  stat->time_to_first_byte = std::chrono::steady_clock::now();
+}
+
```

Comment:
I think these recorded period of time is per worker.  But worker can handle multiple Client object which has its own connection to the server.  So probably we should record them per Client not per worker.